vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,52 +1,76 @@
-# Overview and Objective
- 
-The Entando Process Driven Applications plugin is built to provide a
-rich and full-featured user experience, facilitating the management and
-completion of business processes and automation.
- 
-The general purpose UX layer is created from micro frontends that can be
-utilized for any business process or task engine. The UX layer can serve data via the included Entando integration adapter, or by implementing a set of interfaces on the server side.
- 
-The backend is a Spring Boot microservice that provides a pluggable interface to allow the injection of underlying processes or automation toolkits. The interfaces and steps for creating a new PDA backend implementation are described in further detail in a [tutorial](link).
- 
-Visit the following pages to explore and leverage the Entando
-Process Driven Apps (PDA) plugin.
- 
-- The [Installation](link) tutorial provides step-by-step instructions to install the PDA plugin in a user environment.  
-- The [Configuration](link) tutorial introduces PDA features and capabilities via guided configuration.
-- The [Architecture](link) documentation details the components of the PDA plugin architecture.
-- The [Customization and Extension](link) documentation describes how to customize and build upon the PDA plugin.
-- The [Technical Insights](link) tutorial provides a detailed technical overview of how to add new functionality to a PDA implementation.
- 
-# Installation
- 
-# Configuration
- 
-# Architecture
-This document provides a comprehensive description of the building blocks used to create the PDA plugin. These include the following components:
- 
-- Micro Frontend
+---
+sidebarDepth: 2
+---
+
+# Entando Process Driven Applications Plugin
+
+## Introduction
+
+The Entando Process Driven Applications (PDA) plugin is engineered to provide a rich and full-featured user experience while facilitating the management and completion of business processes and automation.  The solution template includes: 
+
+- A general purpose UX layer that is created from micro frontends and can be implemented for any business process or task engine. The UX layer can serve data via the included Entando integration adapter or by implementing a set of interfaces on the server side.
```

Comment:
A general purpose UX layer created from micro frontends and implemented for any business process or task engine. 

Second sentence: 'via the included Entando integration' makes you pause...it's not wrong but the "in, en, in" sounds makes you kind of pause. Maybe just me

Reply:
i'll take out "that is" but i think it's important to keep "can be" - it's optional, not inevitable

"included Entando integration"? lol now that i think about it, it actually sounds melodic to me :P rewording to avoid that is making the sentence longer :(

Reply:
It does make it melodic. Don't make the sentence longer! Is included needed? It's minor so I wouldn't worry about it.

Reply:
i think we should err on the side of caution and keep it in, just to clarify that the user doesn't have to go through the extra step of procuring or configuring it